astropy/io/fits/diff.py

```diff
@@ -343,7 +357,8 @@ def _report(self):
                           wrapper.fill(ignore_fields))
         self._writeln(u(' Maximum number of different data values to be '
                         'reported: %s') % self.numdiffs)
-        self._writeln(u(' Data comparison level: %s') % self.tolerance)
+        self._writeln(u(' Reltol: %s, Abstol: %s' ) %
+                      (self.reltol, self.abstol))
```

Comment:
Here I'd rather spell out "Relative tolerance:" "Absolute tolerance:"   I don't like cryptic abbreviations when there's plenty of space on the line (even if in this case most people will know what these mean).